src/cogs/mod/views/role.py

```diff
@@ -39,7 +38,7 @@ async def callback(self, interaction: discord.Interaction):
 
 class RoleCancelButton(discord.ui.Button):
     def __init__(self, ctx: Context, *, role: discord.Role, members: typing.List[discord.Member]):
-        super().__init__(timeout=None)
+        super().__init__()
```

Comment:
`discord.ui.Button` don't take `timeout` kwarg :thinking: I don't know how it was working earlier.